.github/workflows/build-verification.yml

```diff
@@ -77,15 +77,15 @@ jobs:
       - name: Setup ZULU_JDK
         uses: actions/setup-java@v5
         with:
-          java-version: 11.0.12
+          java-version: 17.0.16+8
           distribution: 'zulu'
       - name: Set ZULU_JDK environment variable
         run: |
           echo "ZULU_JDK=$JAVA_HOME" >> $GITHUB_ENV
       - name: Setup ZULU_ALT_JDK
         uses: actions/setup-java@v5
         with:
-          java-version: 11.0.13
+          java-version: 17.0.10+7
```

Comment:
Not an issue, just interested if there's a special reason why we're setting a specific patch, instead of simply `17`?

Reply:
We have a test that verifies the behavior of the JdkImageWorkaround for different (minor) versions of the same vendor:
https://github.com/gradle/android-cache-fix-gradle-plugin/blob/main/src/test/groovy/org/gradle/android/JdkImageWorkaroundTest.groovy#L77